recipes/stir/meta.yaml

```diff
@@ -0,0 +1,53 @@
+
+{% set name = "STIR" %}
+{% set version = "v3_0_1" %}
+{% set sha256 = "4f257fa7a4b15c9d6eddf981d80b79fc27a3ed10df4b8184d507d82a22f3596e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/olivierverdier/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [py2k]
+  skip: true  # [win and py<36]
```

Comment:
This means the library won't be installable on Windows with python 3.5...it's an unfortunate consequence of the way conda-build 2 works. Did you kill 3.5 because it took too long to build for both?

Reply:
Yes, Appveyor times out if I build for both versions. (One version takes 45-50 mins to build, and there is a 90 mins limit). Is there another way around this?

Reply:
It might be possible in conda-build 3. We can also merge it for py36 only and then add 35 support in the feedstock, where it runs the jobs separately (and has separate timeouts) for different versions.